GeneratorInterface/GenFilters/python/VBFGenJetFilter_cfi.py

```diff
@@ -49,4 +49,15 @@
   minDeltaEta        = cms.untracked.double(   3.0), # Minimum dijet delta eta
   maxDeltaEta        = cms.untracked.double(99999.)  # Maximum dijet delta eta
 
-)
\ No newline at end of file
+)
+
+vbfGenJetFilterD = cms.EDFilter("VBFGenJetFilter",
+
+  inputTag_GenJetCollection = cms.untracked.InputTag('ak4GenJetsNoNu'),
+
+  leadJetsNoLepMass         = cms.untracked.bool  ( True),  # Require the cut on the mass of the leading jets
```

Comment:
this should probably be False

Reply:
not true, should be True...
Does it work if leadJetsNoLepMass and the other parameters are not specified?

Reply:
If leadJetsNoLepMass is true but no other parameters are specified the code works.
In that case, the events pass the selection whenever there are at least two jets with DR > 0.3 from all the charged hard leptons.

Reply:
If leadJetsNoLepMass is not specified, it is set as false